README.md

```diff
@@ -855,7 +855,8 @@ firewall zones to define the trust level of network connections or interfaces. !
  34. [alim0x](https://github.com/alim0x)
  35. [Spacebody](https://github.com/Spacebody)
  36. [AyoIsaiah](https://github.com/ayoisaiah)
- 37. *Everyone @ reddit.com/r/linux & reddit.com/r/ubuntu & vozforums.com for many suggestions and compliments*
+ 37. [sajattack](https://github.com/sajattack)
```

Comment:
Haha this is fine. I am gonna replace all the list with a link to https://github.com/LewisVo/Awesome-Linux-Software/graphs/contributors. Right now there are about 50 contributors, which makes the list no longer suitable. ;)